test-cases/features/elasticity/scalein-scaleout-elasticity-90-percent.yaml

```diff
@@ -0,0 +1,42 @@
+test_duration: 3000
+prepare_write_cmd: [
+  "cassandra-stress write no-warmup cl=ALL n=300000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1..300000000",
+  "cassandra-stress write no-warmup cl=ALL n=300000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=300000001..600000000",
+  "cassandra-stress write no-warmup cl=ALL n=300000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=600000001..900000000",
+  "cassandra-stress write no-warmup cl=ALL n=250000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=900000001..1200000000",
+  "cassandra-stress write no-warmup cl=ALL n=300000000 -schema 'replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=200 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1200000001..1500000000",
+]
+
+stress_cmd_w: "cassandra-stress write no-warmup cl=ALL n=335544320 -schema 'keyspace=refill_keyspace replication(strategy=NetworkTopologyStrategy,replication_factor=3)' -mode cql3 native -rate threads=300 -col 'size=FIXED(128) n=FIXED(8)' -pop seq=1500000001..1835544321"
```

Comment:
is it intended to do cl=ALL? usually we base on QUORUM as this is what most users do.
btw. what is the purpose of this command? - it is not bound by duration

Reply:
> is it intended to do cl=ALL? usually we base on QUORUM as this is what most users do. 

+1
Since we have RF=3, the disk utilization must be about the same, but CPU utilization and latency will be smaller if we use CL=Quorum.

> btw. what is the purpose of this command? - it is not bound by duration

It uses different keyspace - `refill_keyspace`, so, it adds more data/"disk utilization".